### PR TITLE
watsonx.data L4 Lab: Using IBM Cloud and Amazon S3 Object Storage

### PR DESCRIPTION
**Pull request by MaxShapiro32**
Relates to **[https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/284]**

Content changes:

- Adding lab for watsonx.data L4 Lab: Using IBM Cloud and Amazon S3 Object Storage

### CODE REVIEW

File: content/Watsonx/WatsonxData/202.md [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+3. Click **Create** at the bottom of the **Create a Custom Bucket** tile.
+
+4. Fill out the following for creating a custom bucket:
+  - Enter a unique bucket name. The name must be unique across all IBM COS users, so use `watsonx-data-iceberg-<your initials or unique string>`.
+
+  - Select the **Regional** tile for the Resiliency level.
+  
+  - Click the **Location** dropdown and select the location where you want your data to be stored; this lab guide uses **United States - Dallas (us-south)**, but you can choose whatever location you want.
+
+  **Note: Remember the resiliency and location you chose. You will need to provide this information later.**
```
Comment (r2raman) [inline content/Watsonx/WatsonxData/202.md:58]: enclose all notes in > **Note:** tag

File: content/Watsonx/WatsonxData/202.md [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+
+  - Select the Standard tile for the Storage class.
+
+  - Click the blue **Create bucket** button at the bottom of the page. **Do not change** anything else.
+
+  ![](./images/202/cos-create-bucket-settings.png)
+
+  **Note: If you get the Enable Versioning for your Bucket pop-up, click Maybe Later.**
```
Comment (r2raman) [inline content/Watsonx/WatsonxData/202.md:66]: enclose all notes in > **Note:** tag

File: content/Watsonx/WatsonxData/202.md [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+
+This lab requires that you already have a provisioned IBM watsonx.data environment. This is done in **[Module 1: Environment Setup](/watsonx/watsonxdata/1)**
+
+### Amazon S3 Object Storage
+
+You must have an Amazon Web Services (AWS) account to be able to use Amazon S3 storage. Create an AWS account and get started with S3 **[here](https://aws.amazon.com/pm/serv-s3/)**
+
+> **Note:** As part of the **[AWS Free Tier](https://aws.amazon.com/free/)**, you can get started with Amazon S3 for free. Upon sign-up, new AWS customers receive 5GB of Amazon S3 storage in the S3 Standard storage class; 20,000 GET Requests; 2,000 PUT, COPY, POST, or LIST Requests; and 100 GB of Data Transfer Out each month.
+
+### IBM Cloud Object Storage (COS)
```
Comment (r2raman) [inline content/Watsonx/WatsonxData/202.md:30]: Include a section to to let participants know that, in the class room setting, we will provide object storage detail for IBM CLoud and AWS

**Review by r2raman — CHANGES_REQUESTED** [review]

File: content/Watsonx/WatsonxData/202.md (hunk 1 above)
Comment (MaxShapiro32) [reply content/Watsonx/WatsonxData/202.md]: Fixed

File: content/Watsonx/WatsonxData/202.md (hunk 2 above)
Comment (MaxShapiro32) [reply content/Watsonx/WatsonxData/202.md]: Fixed

File: content/Watsonx/WatsonxData/202.md (hunk 3 above)
Comment (MaxShapiro32) [reply content/Watsonx/WatsonxData/202.md:30]: Added a relevant Note

**Review by r2raman — APPROVED** [review]

**Pull request merged**